Content.Server/StationEvents/Events/IonStormRule.cs

```diff
@@ -124,7 +128,10 @@ protected override void Started(EntityUid uid, IonStormRuleComponent comp, GameR
             }
 
             // generate a new law...
-            var newLaw = GenerateLaw();
+            // SS220 IonStrom Laws rework start
+            // var newLaw = GenerateLaw();
+            var newLaw = PickLaw();
```

Comment:
```suggestion
            var newLaw = Pick(BrickedLaw);
```
Не думаю что нужно создавать отдельный метод для этого (я в комментах потом об этом и сказал).